docs/src/main/docs/tracing/01_tracing.adoc

```diff
@@ -57,3 +57,72 @@ ServerConfiguration.builder()
 ----
 <1> The name of the application (service) to associate with the tracing events
 <2> The endpoint for tracing events, specific to the tracer used, usually loaded from Config
+
+=== Configuring Tracing with Helidon MP
```

Comment:
Shouldn't this be instead a separate document under `microprofile/` ?

Reply:
yes, fixed